vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -53,10 +56,13 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/windows/nanoserver:1809",
             "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
             "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.3.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.4.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.2.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.3.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.4.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.4.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.5.0",
```

Comment:
Currently azuredisk-csi:v1.4.0 is not used but 1.5.0 is used, is it right?

Reply:
Agree, please make sure csi:v1.4.0 is not in production.

Reply:
correct, it's just upgraded to v1.5.0 two weeks ago.